NEWS.md

```diff
@@ -1,5 +1,10 @@
 # easystats 0.5.3
 
+* Following functions have been renamed (old -> new):
+
+  - `CRAN_checks()` -> `check_status_on_cran()`
+  - `on_CRAN()` -> `last_updated_on_cran()`
```

Comment:
you said in #299 you wanted a common prefix, how about `cran_status()` and `cran_last_update()`

Reply:
Hmm, yeah, but I also like function names starting with verbs.

How about:

- `check_status_on_cran()`
- `check_previous_cran_update()`

Reply:
`check_previous_cran_update()` -> I would expect something related to tests and failures, not date of last update. But the original names are fine

Reply:
``` r
library(easystats)
#> # Attaching packages: easystats 0.5.2.6
#> ✔ bayestestR  0.12.1.4     ✔ correlation 0.8.2.3   
#> ✔ datawizard  0.5.1.6      ✔ effectsize  0.7.0.9999
#> ✔ insight     0.18.2.8     ✔ modelbased  0.8.5     
#> ✔ performance 0.9.2.2      ✔ parameters  0.18.2.5  
#> ✔ report      0.5.5.1      ✔ see         0.7.2.2
check_last_cran_update()
#> bayestestR  18.7 weeks (2022-05-02)
#> correlation 4.6 weeks (2022-08-09)
#> datawizard  3.4 weeks (2022-08-17)
#> easystats   1.6 weeks (2022-08-30)
#> effectsize  4.4 weeks (2022-08-10)
#> insight     4.4 weeks (2022-08-10)
#> modelbased  3.3 weeks (2022-08-18)
#> performance 4.4 weeks (2022-08-10)
#> parameters  4.4 weeks (2022-08-10)
#> report      2.7 weeks (2022-08-22)
#> see         4.4 weeks (2022-08-10)
```

<sup>Created on 2022-09-10 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>